dev/com.ibm.ws.st.liberty.gradle/src/com/ibm/ws/st/liberty/gradle/servertype/internal/LibertyGradleJEEPublisher.java

```diff
@@ -125,7 +127,7 @@ public void publishModuleAndChildren(int kind, PublishUnit unit, MultiStatus mSt
      * @param monitor
      */
     protected void publishOnParent(WebSphereServer wsServer, IProject moduleProject, LibertyBuildPluginConfiguration config,
-                                   String gradleCommand, int kind, PublishUnit unit,
+                                   String[] tasks, int kind, PublishUnit unit,
```

Comment:
Should we pass a List instead of an array?

Reply:
I used an array to force the code to create a list or make a new array in order to add to it (since something like LibertyGradleConstants.SKIP_TESTS_ARGS should not be modified.  Also the Gradle tooling API takes arrays.